src/controllers/admin.js

```diff
@@ -151,14 +157,20 @@ adminController.categories.getAll = function(req, res, next) {
 		if (err) {
 			return next(err);
 		}
-
-		categoryData.filter(Boolean).forEach(function(category) {
-			(category.disabled ? disabled : active).push(category);
-		});
-
-		res.render('admin/manage/categories', {
-			active: active,
-			disabled: disabled
+        
+        plugins.fireHook('filter:admin.categories.getAll', {req: req, res: res, categories: categoryData}, function(err, data) {
+			if (err) {
+				return next(err);
+			}
+            
+			data.categories.filter(Boolean).forEach(function(category) {
+                (category.disabled ? disabled : active).push(category);
+            });
+
+            res.render('admin/manage/categories', {
+                active: active,
+                disabled: disabled
+            });
```

Comment:
![2015-04-01 18 11 32](https://cloud.githubusercontent.com/assets/465108/6944063/934eac4a-d89a-11e4-819f-40abd611a924.png)
a sublime text 3 and brackets so good